lms/djangoapps/courseware/testutils.py

```diff
@@ -197,6 +199,13 @@ def test_success_enrolled_staff(self, default_store, mongo_calls):
             with check_mongo_calls(mongo_calls):
                 self.verify_response()
 
+    def get_success_enrolled_staff_mongo_count(self):
```

Comment:
See lms/djangoapps/lti_provider/tests/test_views.py below. I didn't get all the way to the root cause here, but given that it is old mongo only (and lti only), I didn't think it was worth spending time on it.